### PR TITLE
Add Dogium Farm

### PR DESCRIPTION
**Pull request by pepperstepper**
**NOTE**

1. The protocol is usually listed within 24 hours of merging the PR
2. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
3. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
4. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
5. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

##### Twitter Link:
https://twitter.com/PolyWantsAFarm

##### List of audit links if any:
https://paladinsec.co/projects/PolyWantsACracker/

##### Website Link:
https://lithium.dog

##### Current TVL:
150kusd

##### Chain:
dogechain

##### Short Description (to be shown on DefiLlama):
PolyWantsADoge is a yield farm and yield maximizer on the Dogecoin Network. It is a fork of the PolyWantsACracker codebase by the original team. Tokens can be swapped for Janis DEX presale credit.

##### Token address and ticker if any:
 0x55BD2a3904C09547c3A5899704f1207eE61878Be, DOGIUM

##### Category (full list at https://defillama.com/categories) *Please choose only one:
[Liquid Staking](https://defillama.com/protocols/Liquid%20Staking)

##### forkedFrom (Does your project originate from another project):
PolyWantsACracker
https://polywantsacracker.farm/#/

##### methodology (what is being counted as tvl, how is tvl being calculated):
Staked assets USD value

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/dogium-farm exports TVL: 
        
 
 
    dogechain-pool2           0
    dogechain-staking         0
    dogechain                 0
    dogechain-masterchef      0
    pool2                     0
    staking                   0
    masterchef                0
    
    total                    0

**Comment by pepperstepper** [comment]
Hi Are there any blockers on llamas end?

File: projects/dogium-farm/index.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const { masterChefExports } = require("../helper/masterchef");
```
Comment (g1nt0ki) [inline projects/dogium-farm/index.js:1]: try 

    const { masterchefExports } = require("../helper/unknownTokens");

    const dogiumToken  = "0x55bd2a3904c09547c3a5899704f1207ee61878be";
    const masterchef = "0x579BACCd9DdF3D9e652174c0714DBC0CD4700dF2";

    module.exports = masterchefExports({ chain: 'dogechain', masterchef, useDefaultCoreAssets: true, coreAssets: [dogiumToken],  })

**Comment by llamatester** [comment]
The adapter at projects/dogium-farm exports TVL: 
        
 
 
    dogechain-staking         0
    dogechain-pool2           0
    dogechain-masterchef      0
    dogechain                 0
    staking                   0
    pool2                     0
    masterchef                0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/dogium-farm exports TVL: 
        
 
 
    dogechain                 193.96 k
    dogechain-staking         13.77 k
    dogechain-pool2           21.51 k
    staking                   13.77 k
    pool2                     21.51 k
    
    total                    193.96 k

**Comment by pepperstepper** [comment]
Okay try that!

**Pull request merged**